Makefile

```diff
@@ -158,6 +158,12 @@ binaries:
 	tar -czvf boltz-client-linux-arm64-v$(VERSION).tar.gz bin/linux_arm64
 	sha256sum boltz-client-*.tar.gz bin/**/* > boltz-client-manifest-v$(VERSION).txt
 
+gdk-source: submodules
+	cd gdk && git checkout release_$(GDK_VERSION) && git apply ../gdk.patch
+
+build-gdk-builder: gdk-source
+	docker buildx build --push -t boltz/gdk-ubuntu-builder:latest -t boltz/gdk-ubuntu-builder:$(GDK_VERSION) $(DOCKER_ARGS) -f ./gdk/docker/ubuntu/Dockerfile ./gdk
```

Comment:
`gdk.Dockerfile` is still saying `FROM michael1011/gdk-ubuntu-builder:$GDK_VERSION AS builder`